### DISCUSSION TITLE
TypeError in Middleware File

### DISCUSSION THREAD

**Discussion by for-Ry-in-range** [clerk/javascript#3534] [Support request - DO NOT OPEN HERE]
Hi, I'm getting this TypeError on my middleware.ts file and I haven't been able to find a solution to this.
![image](https://github.com/clerk/javascript/assets/108443873/1b06932d-ebdb-413a-bbab-3c702deb0f05)

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
